### PR TITLE
remove npm

### PR DESCRIPTION
**Pull request by J-Thompson12**
## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: tests-solidity/README.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ Increasingly difficult tests are provided:
 
 **Prerequisite**: in the repo's root, run `make install` to install the `ethermintd` and `ethermintcli` binaries. When done, come back to this directory.
 
-**Prerequisite**: install the individual solidity packages. They're set up as individual reops in a yarn monorepo workspace. Install them all via `yarn install`.
+**Prerequisite**: install the individual solidity packages. They're set up as individual reops in a npm monorepo workspace. Install them all via `npm install`.
+
+**Prerequisite**: install truffle using command `npm install -g truffle`
```
Comment (sohkai) [inline tests-solidity/README.md]: As the original contributor for these, I don't have a strong preference between yarn and npm, but to bring awareness on my choice:

1. I would strongly push against advocating for any npm module to be installed globally to run these tests (or really, to do any local-repo development)
2. Yarn will be much, much faster at installing these test sub-packages, especially noticeable on CI environments
3. If we do use npm, let's commit the `package-lock.json` :).
4. Note that `npm` does not come with its own monorepo capabilities, and so you would have to either script, use `lerna`, or go into each of the separate packages to install them before they can be run (since they require other dependencies than just truffle).

Comment (fedekunze) [reply tests-solidity/README.md]: I was under the impression that `yarn` was less secure (I'm possibly wrong at this since the last time I used it continuously was in 2017-18), couldn't find any resources that backed this. Performance-wise, yarn is actually better than npm (see [here](https://stackshare.io/stackups/npm-vs-yarn) and [here](https://www.keycdn.com/blog/npm-vs-yarn#speed)). There's a community comparison between the two packages [here](https://www.slant.co/versus/4234/17851/~npm_vs_yarn).

In summary, I think we can undo the changes from this PR and use yarn everywhere (including docs) for the purpose of standardization.

**Review by fedekunze — COMMENTED** [review]
I can't build the docs on http://localhost:8080/ using `make docs-serve`. I think the yarn methods don't require the `run` command: `npm run serve` -> `yarn serve`. Or maybe the yarn.lock file is wrong? afaik there's a cmd to migrate to yarn from npm 

Ref: https://nodejs.dev/learn/the-package-json-guide

**Comment by J-Thompson12** [comment]
Okay not sure why the existing entities were not working but it works now

**Review by fedekunze — APPROVED** [review]
tested ACK
